db/nft.go

```diff
@@ -260,7 +260,8 @@ func GetCollector(conn *pgxpool.Conn, q QueryCollectorRequest) (res QueryCollect
 		FROM iscn as i
 		JOIN nft_class as c ON i.iscn_id_prefix = c.parent_iscn_id_prefix
 		JOIN nft AS n ON c.class_id = n.class_id
-		WHERE i.owner = $1
+			AND ($4::text[] IS NULL OR n.owner != ALL($4))
+		WHERE $1 = '' OR i.owner = $1
```

Comment:
Are these 2 lines order reversed? Should it be something like below?
```
		WHERE ($1 = '' OR i.owner = $1)
			AND ($4::text[] IS NULL OR n.owner != ALL($4))
```

Reply:
I refer here: https://github.com/likecoin/likecoin-chain-tx-indexer/blob/22f9a79839dc4fda86e269418a259f28aa553925/db/nft.go#L81
Or should the order be reversed here?

Reply:
Turns out the planner gives the same plan, so I guess it's OK